src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -96,19 +97,19 @@ private boolean checkIfExecute()
     {
         setDelay(1);
 
-        if(!job.hasWorkOrder())
+        if (!job.hasWorkOrder())
         {
             return true;
         }
 
         WorkOrderBuild wo = job.getWorkOrder();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'wo' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)